arcade-services.sln

```diff
@@ -140,554 +140,384 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.Internal.L
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DncEng.DeployServiceFabricCluster", "src\Microsoft.DncEng.DeployServiceFabricCluster\Microsoft.DncEng.DeployServiceFabricCluster.csproj", "{6307FF01-47DB-4EE6-86FF-9E78D8832D37}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.DncEng.Configuration.Bootstrap", "src\Microsoft.DncEng.Configuration.Bootstrap\Microsoft.DncEng.Configuration.Bootstrap.csproj", "{2D72FBB5-9F29-40D5-9206-92A814DF47EC}"
```

Comment:
Can we really remove this?

Reply:
I'm guessing merge error?

Reply:
Yea, this looks like a merge issue.

Reply:
Yeah, I had the same merge conflict. :-) sln files are the worst because of the "EndProject" lines, nothing can merge them automatically without screwing them up.

Reply:
This file change messes my entire project. It always screws up my commits. :(